packages/turbo/README.md

```diff
@@ -252,6 +252,14 @@ Session component has been deprecated. To use multiple [sessions](https://github
 
 ## Other utilities
 
-### `clearSnapshotCacheForAllSessions`
+### `getRegisteredSessionHandles`
```

Comment:
Nitpick, but is `Registered` needed here? We don't have any session handles that are not registered.